CHANGELOG.md

```diff
@@ -39,6 +39,11 @@
   [alvarhansen](https://github.com/alvarhansen)
   [#2746](https://github.com/realm/SwiftLint/issues/2746)
 
+* Don't trigger `file_types_order` violations in files only containing 
+  extensions.
```

Comment:
Please add two whitespaces at the end of this line for correct GitHub line breaks.

Reply:
Done, thanks!